appveyor.yml

```diff
@@ -0,0 +1,24 @@
+# Configuration file for AppVeyor continuous integration on Windows
+# https://ci.appveyor.com/project/josef-pkt/statsmodels
+
+#os: Windows Server 2012
+
+platform:
+  - x64
+
+install:
+  - ps: Invoke-WebRequest "https://bootstrap.pypa.io/ez_setup.py" -OutFile "c:/ez_setup.py"
+  - ps: Invoke-WebRequest "https://bootstrap.pypa.io/get-pip.py" -OutFile "c:/get-pip.py"
+  - C:\Python27\python.exe c:\get-pip.py
+  - C:\Python27\Scripts\pip.exe install conda
+  - C:\Python27\Scripts\conda.exe init
+  - C:\Python27\Scripts\conda.exe create -n py3 --yes --quiet python=3 numpy scipy cython pandas pip nose patsy
+
+build_script:
+  - C:\Python27\envs\py3\python.exe setup.py install
+
+test_script:
+  - cd C:\Python27\envs\py3\Scripts
+#  - activate.bat
+  - SET PATH=%PATH%;C:\Python27\envs\py3\Scripts
```

Comment:
If you like this way, I think it should be

```
SET PATH=C:\Python27\envs\py3\;C:\Python27\envs\py3\Scripts;%PATH%;
```

So that py3 is before the default Python on the path.